docs/reference/docs/bulk.asciidoc

```diff
@@ -1,28 +1,36 @@
 [[docs-bulk]]
 === Bulk API
+++++
+<titleabbrev>Bulk</titleabbrev>
+++++
 
-The bulk API makes it possible to perform many index/delete operations
-in a single API call. This can greatly increase the indexing speed.
+Performs multiple indexing or delete operations in a single API call. 
+This reduces overhead and can greatly increase indexing speed.
 
-.Client support for bulk requests
-*********************************************
-
-Some of the officially supported clients provide helpers to assist with
-bulk requests and reindexing of documents from one index to another:
-
-Perl::
+[source,console]
+--------------------------------------------------
+POST _bulk
+{ "index" : { "_index" : "test", "_id" : "1" } }
+{ "field1" : "value1" }
+{ "delete" : { "_index" : "test", "_id" : "2" } }
+{ "create" : { "_index" : "test", "_id" : "3" } }
+{ "field1" : "value3" }
+{ "update" : {"_id" : "1", "_index" : "test"} }
+{ "doc" : {"field2" : "value2"} }
+--------------------------------------------------
 
-    See https://metacpan.org/pod/Search::Elasticsearch::Client::5_0::Bulk[Search::Elasticsearch::Client::5_0::Bulk]
-    and https://metacpan.org/pod/Search::Elasticsearch::Client::5_0::Scroll[Search::Elasticsearch::Client::5_0::Scroll]
+[[docs-bulk-api-request]]
+==== {api-request-title}
 
-Python::
+`POST /_bulk`
+`POST /<index>/_bulk`
```

Comment:
Currently, these display on the same line: http://elasticsearch_47479.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/docs-bulk.html#docs-bulk-api-request
```suggestion

`POST /<index>/_bulk`
```